packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.base.tsx

```diff
@@ -79,6 +80,7 @@ export class TooltipHostBase extends BaseComponent<ITooltipHostProps, ITooltipHo
         {...{ onBlurCapture: this._hideTooltip }}
         onMouseEnter={this._onTooltipMouseEnter}
         onMouseLeave={this._onTooltipMouseLeave}
+        onKeyDown={this._onTooltipKeyDown}
```

Comment:
does this onKeyDown need to be on both the div and the Tooltip and the callout?

Reply:
Actually it looks like we might just need it on the outermost div. I'll push the changes